decimal.go

```diff
@@ -1790,6 +1799,20 @@ func (d Decimal) MarshalJSON() ([]byte, error) {
 	return []byte(str), nil
 }
 
+// MarshalYAML implements the yaml.Marshaler interface.
```

Comment:
Please try


```suggestion
// MarshalYAML implements the [yaml.Marshaler] interface.
```

I think it's the right syntax for using godoc 1.19 links syntax